diff.js

```diff
@@ -41,7 +43,7 @@ function splitByFile( diff ) {
 
 		if ( line.charAt( 0 ) === "d" ) {
 			isEmpty = false;
-			filename = line.replace( /^diff --git a\/(\S+).*$/, "$1" );
+			filename = line.replace( /^diff --[git|cc] a\/(\S+).*$/, "$1" );
```

Comment:
This doesn't seem right. You've created a character set instead of a group. `--cc` diffs also don't have the `a/` path prefix. Using `/^diff --(?:cc |git a\/)(\S+).*$/` seems to do what you want. Can you confirm?